lib/features/settings/widgets/language_settings_view.dart

```diff
@@ -0,0 +1,70 @@
+import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+
+import "../../../../config/ui_config.dart";
+import "../../../../theme/app_theme.dart";
+import "../../../utils/context_extensions.dart";
+
+class LanguageDialog extends ConsumerWidget {
+  const LanguageDialog();
+
+  static Future<String?> show(BuildContext context) async {
+    return showDialog<String>(context: context, builder: (context) => const LanguageDialog());
+  }
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return AlertDialog(
+      backgroundColor: context.colorTheme.whiteSoap,
+      contentPadding: const EdgeInsets.all(AboutUsConfig.spacerHeight),
+      content: StatefulBuilder(
+        builder: (context, setState) {
+          String selectedLanguage = context.localize.localeName;
+          final languages = [
+            {"code": "pl", "name": "Polski 🇵🇱"},
+            {"code": "en", "name": "English 🇬🇧"},
```

Comment:
```suggestion
            {"code": "pl", "name": "🇵🇱 Polski"},
            {"code": "en", "name": "🇬🇧 English"},
```

Idk WDYT - i like this better this way but some may disagree